app/views/setup/welcome.html.slim

```diff
@@ -25,6 +25,23 @@ h1 Initial CaaS Platform Configuration
             = f.check_box "tiller", { checked: @tiller }, "true", "false"
             | Install Tiller (Helm's server component)
 
+  .panel.panel-default
+    .panel-heading.clearfix
+      h3.panel-title Registry settings
+      .pull-right
+        = label_tag :registry_settings_toggle, nil, class: "btn btn-default btn-sm js-toggle-registry-settings-btn", data: {toggle: "collapse", target: "#registry-settings-panel"}
+          | Show
+    #registry-settings-panel.panel-collapse.collapse
+      .panel-body
+        p Local insecure registries are highly discouraged and should be avoided. However, in some setups it might be necessary. You can add an insecure registry here.
+        .form-group.registry-form
+          = f.label :insecure_registries, "List of external insecure Docker registries (optional)"
+          - if @insecure_registries.any?
+            - @insecure_registries.each do |registry|
+              = render "setup/registry", f: f, registry: registry
+          - else
+            = render "setup/registry", f: f
```

Comment:
Could we pass registry: DockerRegistry.new instead of nothing to this partial when there's no registries to avoid the ternary conditions inside the partial? Just wondering...